File: Python/BEGINNER/1983 The Chosen.py
# SUBPROGRAM
# MAIN PROGRAM
highest = 0
students = int(input())
while students != 0:
    registration, grade = map(float, input().split())
    int(registration)
    if highest < grade:
        highest = grade
        approved = registration
    students -= 1
if highest < 8:
    print("Minimum note not reached")
else:
    print("%d" % approved)

File: C/BEGINNER/1983 The Chosen.c
#include <stdio.h>

int main() {
    int students;
    long int registration, approved;
    float grade, highest = 0;
    scanf("%d", &students);
    while (students != 0)
    {
        scanf("%ld %f", &registration, &grade);
        if (highest < grade)
        {
            highest = grade;
            approved = registration;
        }
        students -= 1;
    }
    if (highest < 8)
        printf("Minimum note not reached\n");
    else
        printf("%ld\n", approved);
    return 0;
}
